README.md

```diff
@@ -1,14 +1,14 @@
-## Websky Intern's Capstone Project
+# Websky Intern's Capstone Project
 
-### Background
-There are users who own a Wear OS or Android devices that have an iPhone. 
+## Background
+There are users who own a Wear OS or Android devices that have an iPhone.
 Currently, there is no platform outside of the Android watch play store that allows these users to browse and download applications
-directly to their device. This web platform will eventually be a feature on the existing Google play webstore, making the app store 
-more accessible and user friendly. 
+directly to their device. This web platform will eventually be a feature on the existing Google play webstore, making the app store
+more accessible and user friendly.
 
-### Objective
+## Objective
 Our objective is to create a platform on the Google Play webstore for Wear OS users to easily access and download applications for their devices.
 
-### Design Details 
+## Design Details
 Our design details and timeline are covered further in the design doc.
-Link: https://docs.google.com/document/d/16WTpYj0lHeBMeNxVi5iF3h0UVU-DOMzJVR9vFNM_I-g/edit?usp=sharing
+Link: <https://docs.google.com/document/d/16WTpYj0lHeBMeNxVi5iF3h0UVU-DOMzJVR9vFNM_I-g/edit?usp=sharing>
```

Comment:
You can change this to `[Link:](https://docs.google.com/document/d/16WTpYj0lHeBMeNxVi5iF3h0UVU-DOMzJVR9vFNM_I-g/edit?usp=sharing)`